docs/_content/overige/technisch/zds-vs-zgw.md

```diff
@@ -7,27 +7,22 @@ De Zaak- en Documentservices (ZDS) wijken af van de Zaakgericht werken (ZGW)
-9. De ZGW APIs worden net als ZDS beheerd en onderhouden door VNG, zij het in
-   een andere vorm.
+[Zaak-Documentservices](https://www.gemmaonline.nl/index.php/Zaak-_en_Documentservices) | [ZGW APIs](https://ref.tst.vng.cloud/standaard/index)\*
+--- | ---
+SOAP/XML gebaseerd	                                                                        | REST/JSON gebaseerd
+Gebaseerd op het informatiemodel RGBZ1	                                                    | Gebaseerd op informatiemodel RGBZ2 waar van toepassing
+Verdere invulling/uitwerking van de basisberichten uit StUF ZKN	                            | Aparte calls voor aparte resources
+Gebaseerd op uitwisselen van informatie, gestructureerd volgens het informatiemodel [RGBZ 1](https://www.gemmaonline.nl/index.php/Informatiemodel_Zaken_(RGBZ))	| Gebaseerd op operaties op resources die leven in het referentiecomponent
+Ondersteunen van de functies van een zaaksysteem zoals beschreven in [GEMMA 1](https://www.gemmaonline.nl/index.php/Overzicht_GEMMA_1_referentiecomponenten)	            | Ondersteunen van functies van de referentiecomponenten zoals beschreven in [GEMMA 2](https://www.gemmaonline.nl/index.php/Overzicht_generieke_referentiecomponenten_GEMMA_2)
+Kent alleen de functionele acties die uitgewerkt zijn in de standaard.	                    | Alle functionele acties die mogelijk zijn met de CRUD functionaliteit zijn mogelijk. De ZGW API's zijn microservices die gecombineerd kunnen worden om een functionele behoefte in te vullen.
+Betrokkenen en/of gerelateerde objecten kunnen geheel of gedeeltelijk, in ieder geval met matchinggegevens doorgegeven worden. Deze worden veelal in het zaaksysteem opgeslagen.	                                                             | een relatie naar een andere resource is een verwijzing naar die resource. Die resource wordt opgeslagen in de betreffende registratie. Dat kan een zaakregistratie (ZRC) zijn maar ook een andere (met een eigen API). 
+bestaan voornamelijk uit a-synchrone services	                                            | bestaan uit alleen maar synchrone services
+Een [GEMMA 1](https://www.gemmaonline.nl/index.php/Overzicht_GEMMA_1_referentiecomponenten) Zaaksysteem bevat ook (generieke) zaakafhandelfunctionaliteit. Deze hoeft niet per se via ZDS te communiceren met het eigen zakenmagazijn  | Een [GEMMA 2](https://www.gemmaonline.nl/index.php/Overzicht_generieke_referentiecomponenten_GEMMA_2) [Zaakregistratiecomponent](https://www.gemmaonline.nl/index.php/GEMMA2/0.9/id-a97b6545-d5a7-485d-9b13-3ce22db5b9cf) registreert alleen zaken en (verwijzingen naar) gerelateerden. Afhandelen gebeurt in een [zaakafhandelcomponent](https://www.gemmaonline.nl/index.php/GEMMA2/0.9/id-f2dfbd0b-9d36-405c-bdbe-827f3296de29) (ZAC). Communicatie vindt plaats via de ZGW API's
+Zaak- Documentservices is gemaakt om taakspecifieke applicaties te koppelen aan een zaaksysteem. | ZGW API's zijn gemaakt om zaakgericht werken te ondersteunen
```

Comment:
Jij weet dit beter dan ik maar is het speciaal gemaakt voor "taakspecifieke applicaties"? Het is toch vooral dat het een "eenvoudigere laag op StUF-ZKN" is?

Reply:
Eigenlijk heb ik het ook iets te kort door de bocht beschreven. Officieel is ZDS (par 1.3 UItgangspunten en reikwijdte, eerste bullet):
* De services ondersteunen de meest gebruikte basisfunctionaliteit voor de 2-weg applicatie-applicatiekoppeling tussen een DMS en ZS onderling en de koppelingen van een DMS en ZS met andere systemen voor het ontsluiten en muteren van zaakgegevens en zaakdocumenten;

Je kunt ZDS dus ook gebruiken voor bijvoorbeeld zaakbevragingen of om een zakenmagazijn over te zetten naar een ander zakenmagazijn. Iets wat ook kan met de ZGW API's.

Wat ik wilde aangeven is dat het hele proces van zaakgericht werken, dus inclusief het opvragen van zaaktypen etc. in de ZTC, uitgewerkt is in de ZGW API's. In ZDS is slechts het meest gebruikte deel uitgewerkt vanuit het gezichtspunt van de TSA.